app/channel_test.go

```diff
@@ -390,30 +394,6 @@ func TestAddUserToChannelCreatesChannelMemberHistoryRecord(t *testing.T) {
 	assert.Equal(t, groupUserIds, channelMemberHistoryUserIds)
 }
 
-/*func TestRemoveUserFromChannelUpdatesChannelMemberHistoryRecord(t *testing.T) {
```

Comment:
any reason this is commented out and not deleted?

Reply:
Maybe someone left there for some reason but since all the code is in the version control system I don't see any reason to have it there. Commend code is dead code :D

Reply:
@ethervoid looks like it's this change that commenting out this test, do you uncomment then or maybe remove this?

Reply:
The test was commented [here](https://github.com/mattermost/mattermost-server/pull/9889) and seems to be not needed or causing issues. Given that is testing code from 2018, I'd remove it instead of using it

Reply:
oh shoot, i was seeing the red highlight as something being modified rather than being removed 🤦‍♂ my bad... all good here 👍 :)